font-types/src/version.rs

```diff
@@ -13,7 +13,7 @@ pub struct Version16Dot16(u32);
 /// represented as a `major_version`, `minor_version` pair. This type encodes
 /// those as a single type, which is useful for some of the generated code that
 /// parses out a version.
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
```

Comment:
So I'm way late on this, but the pattern we use elsewhere is to explicitly compute the version, using the `#[compile]` annotation. As written, it would be easy to accidentally write out an `avar` table with the version (0, 0), which is invalid.